src/web/app/components/copy-session-modal/copy-session-modal.component.html

```diff
@@ -17,8 +17,8 @@ <h5 class="modal-title">
       </div>
       <div class="form-check" *ngFor="let course of courseCandidates">
         <label class="form-check-label">
-          <input type="radio" name="copySessionChooseCourse" class="form-check-input"
-                 [(ngModel)]="copyToCourseId" [value]="course.courseId">
+          <input type="checkbox" name="copySessionChooseCourse" class="form-check-input"
```

Comment:
You should also bind the `[checked]` property?

Reply:
I'm not sure what the purpose of the binding would be? Or how to implement it with the current design approach even haha

Reply:
If not bind, what will control the status of the checkbox?

Reply:
Hmm I thought that controlling the status of the checkbox wouldn't be necessary, since it's initialised to unchecked? Then when someone clicks it for the first time and triggers the `change` that's when the `set` logic takes over

Reply:
Let me ask you another way around: Where should I make the change if I want to programmatically control whether the checkbox is checked or not?

Reply:
> Let me ask you another way around: Where should I make the change if I want to programmatically control whether the checkbox is checked or not?

`[checked]="copyToCourseSet.has(course.courseId)"`?

Reply:
That's why I ask to add the bind.